### DISCUSSION TITLE
6.4.8

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#616] [公告]
## 错误修复

* 修复通过 WebUI 删除城市规则时无法删除的问题
* 撤销有关分代 ZGC 的更改，用户报告 ZGC 在部分系统上似乎不能很好的工作，回退到 G1GC

## Docker 用户

DockerHub: `ghostchu/peerbanhelper:v6.4.8`
阿里云国内镜像加速: `registry.cn-hangzhou.aliyuncs.com/ghostchu/peerbanhelper:v6.4.8`

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.4.8'>6.4.8</a>.</em>

**Comment by Mishasama** [comment]
GC的问题直接弄个设置，让用户自己选择；或者直接做个blacklist就好了吧？
